### ISSUE TITLE
Ghosty is not accessible with VoiceOver

### ISSUE THREAD

**Issue by sempruijs** [ghostty-org/ghostty#3520]
None of the text is read out loud. I cannot hear what is on the screen nor feel it with braille.

Steps to reproduce:
1. download ghostty from the ghostty.org website.
2. open ghostty
3. press ```cmd F5``` to turn on VoiceOver
4. Try to navigate ghosty with your eyes closed

I understand if this is out of scope for this project but I think it's good to mention.

**Comment by mitchellh** [comment]
Dupe #2351
